cadquery/cq.py

```diff
@@ -1376,12 +1385,19 @@ def polarArray(
 
         # Add additional elements
         for i in range(1, count):
-            phi = math.radians(startAngle + (angle * i))
-            x = radius * math.cos(phi)
-            y = radius * math.sin(phi)
-            points.append((x, y))
+            phi_deg = startAngle + (angle * i)
+            phi = math.radians(phi_deg)
+            x = radius * math.sin(phi)
```

Comment:
Was the code wrong before when `x` was `cos` and `y` was `sin`?

Reply:
I changed the sign convention here: 0 angle maps to (0,r) and the points are distributed clockwise. I think that this is more intuitive but I'm open for discussion.

Reply:
I'm fine with that, just wanted to be sure.